dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -154,13 +156,26 @@ public class FutureEMBuilder extends CompletableFuture<EntityManagerBuilder> imp
      * @param entityClass entity class.
      */
     @Trivial
-    void add(Class<?> entityClass) {
+    void addEntity(Class<?> entityClass) {
         if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
-            Tr.debug(this, tc, "add: " + entityClass.getName());
+            Tr.debug(this, tc, "FutureEMBuilder@" + Integer.toHexString(hashCode()) + " addEntity: " + entityClass.getName());
 
         entityTypes.add(entityClass);
     }
 
+    /**
+     * Adds a Repository interface represented by this FutureEmBuilder
+     *
+     * @param repositoryInterface repository interface
+     */
+    @Trivial
+    void addRepositoryInterface(Class<?> repositoryInterface) {
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
+            Tr.debug(this, tc, "FutureEMBuilder@" + Integer.toHexString(hashCode()) + " addRepositoryInterface: " + repositoryInterface.getName());
```

Comment:
Similar to above, the following should be including the hexadecimal hash code already:
```suggestion
            Tr.debug(this, tc, "addRepositoryInterface: " + repositoryInterface.getName());
```